### PR TITLE
Mvaliev 155 security backends detection

### PR DESCRIPTION
**Pull request by user2589**

### CODE REVIEW

**Comment by user2589** [comment]
[Screenshot](http://imgur.com/gzZGLbq)

**Comment by rezniknn** [comment]
Ship it

**Comment by mansuleman** [comment]
i couldn't see the screenshot provided by @user2589. Even if it is the case, this kind of design is not confirmed by the customer. Another version of design is accepted and should be used. Why is this commit inside security at all?

**Comment by mansuleman** [comment]
Please explain how to test API calls with token.

**Comment by user2589** [comment]
@mansuleman, API token usage is described in plain English right in this pull request. Please read it

**Comment by mansuleman** [comment]
- above discussions about API are broader, out of scope of this pull request.
- the API token usage seems clear, more or less, but I don't have time to test in the next couple days.
- the customer didn't reply about UI, so

If you want quicker, feel free to merge for your convenience. I think we'll be able to resolve questions later.
